librz/include/rz_debug.h

```diff
@@ -465,8 +465,10 @@ RZ_API bool rz_debug_select(RzDebug *dbg, int pid, int tid);
 RZ_API RzDebugPid *rz_debug_pid_new(const char *path, int pid, int uid, char status, ut64 pc);
 RZ_API RzDebugPid *rz_debug_pid_free(RzDebugPid *pid);
 RZ_API RzList /*<RzDebugPid *>*/ *rz_debug_pids(RzDebug *dbg, int pid);
+#ifdef __linux__
 RZ_API RZ_OWN RzList /*<RzList *>*/ *get_pid_thread_list(RZ_NONNULL RzDebug *dbg, int main_pid);
 RZ_API ut64 get_linux_tls_val(RZ_NONNULL RzDebug *dbg, int tid);
+#endif
```

Comment:
instead of doing this, define this inside the `.c` file and have an `#else` which defined some functions which returns `0/NULL`.

Reply:
Hey I have updated this code path! Can you please review